CONTRIBUTING.md

```diff
@@ -24,9 +24,9 @@ The easiest way to help out is to log any issues you find in the books. Unclear
 all kinds, even better ways to present something -- just go to the [issues page][].
 
 **Before creating a new issue**, please review existing issues to see if someone has already
-submitted the same one. Odds are you're not the first to encounter something, so a little quick
+submitted the same one. Chances are you're not the first to encounter something, so a little quick
 research can save everyone some hassle. It's also a good idea to verify that problems still exist in
-the `development` branch when creating new issues.
+the `dev-xxxxx` branch when creating new issues.
```

Comment:
"in the active development branch, which will be one of `dev-major`, `dev-minor`, or `dev-patch`, when creating new issues

Reply:
Changed to "still exist in the development branch (which will be one of `dev-patch`, `dev-minor` or `dev-major`) when creating new issues."